install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml

```diff
@@ -208,6 +214,11 @@
         - name: istio-certs
           mountPath: /etc/certs
           readOnly: true
+{{- end }}
+{{- if $.Values.telemetry.adapterSecret.enabled }}
+        - name: adapter-secret
+          mountPath: /var/adapter/secret
```

Comment:
Is this the typical location for secrets ? What does k8s use for its tokens ? 

As I mentioned earlier - something in mixer will need to look for this path, I can't see how it can possibly work without it. Or is it in the adapter-specific config ? 

Maybe /var/run/secrets/mixer.istio.io ( to match the convention used by k8s ) ?

Reply:
+1, k8s sa default jwt token path is ```/var/run/secrets/kubernetes.io/serviceaccount/token```, and we mount trustworthy jwt to ```/var/run/secrets/tokens/istio-token``` (https://github.com/istio/istio/blob/master/pilot/pkg/model/authentication.go#L46)

Reply:
Updated path to /var/run/secrets/istio.io/telemetry/adapter. Does this make sense to you?

@costinm The secret is used by adapter, via adapter specific config.

Reply:
Love the new path ! We should have all other secrets follow this convention ( gateway, citadel mounts, etc ) !